tests/azure-sdk-for-cpp/Dockerfile-ubuntu

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
```

Comment:
I don't see where this file was used?

Reply:
This file is not currently used, but is a part of this PR since we plan on extending our support to ubuntu in the future and this can be enabled as a part of that effort with minimum additional work